### ISSUE TITLE
cleanup.sh fails on commit

### ISSUE THREAD

**Issue by sjsivert** [GeeWee/reshaper-pre-commit-hook#6]
OS: Arch linux

Output of install-git-hook.sh
```
Cleaning up old versions
Extracting into ./.git/hooks/resharper
Adding pre-commit hook
Marking as executable
Cleaning up...
```

Output when commiting

```
Staged files .vscode/launch.json
.vscode/tasks.json
Purefolio-backend/DatabaseContext.cs
Purefolio-backend/Models/Models.cs
Purefolio-backend/Program.cs
Purefolio-backend/Startup.cs
Purefolio-backend/WeatherForecast.cs
Include string: .vscode/launch.json;.vscode/tasks.json;Purefolio-backend/DatabaseContext.cs;Purefolio-backend/Models/Models.cs;Purefolio-backend/Program.cs;Purefolio-backend/Startup.cs;Purefolio-backend/WeatherForecast.cs
Formatting files...
JetBrains Cleanup Code 2019.3.1
Running in 64-bit mode, .NET runtime 4.0.30319.42000 under Unix 5.8.7.1
Unable to find target solution file in path /home/archie/code/Purefolio-backend/OAI.sln
"sh ./.git/hooks/resharper/cleanupcode.sh --profile="Built-in: Reformat Code" ./OAI.sln --include="$INCLUDE_STRING"" command filed with exit code 1.

```

I was unable to locate what went wrong by looking at the source code. Do you have any suggestions?

**Comment by GeeWee** [comment]
You're absolutely right, I've made a mistake that should've been fixed.

After installing the hook try to go into the .git/hooks/pre-commit hook and change every occurrence (there should be 3) of `./OAI.sln` to `./YourSolutionFile.sln`

Does that solve it?

**Comment by JonRoosevelt** [comment]
Though @NikZy didn't reply, I had the same issue and it solves it.

I've submitted a PR to fix the issue

https://github.com/GeeWee/reshaper-pre-commit-hook/pull/7

**Comment by GeeWee** [comment]
Thanks a bunch Jon
Closed via #7
